File: set163/b.py
getInput = lambda: int(input().split()[0])

def solve(n, nums):
    for i in range(n):
        if nums[i] == 1:
            nums[i] += 1
    for i in range(n - 1):
        while nums[i + 1] % nums[i] == 0:
            nums[i + 1] += 1
    for i in range(n):
        print(nums[i], end=" ")
    print("")

i = getInput()
for _ in range(i):
    n = getInput()
    nums = list(map(int, input().split()))
    solve(n, nums)

File: set163/b.cpp
#include <bits/stdc++.h>
using namespace std;

// Not Dividing
 
void solve(int n, vector<int> nums){
    int count = 0;
    for (int i = 0; i < n; i++) {
        if (nums[i] == 1) {
            nums[i]++;
        }
    } 
    for (int i = 0; i < n - 1; i++) {
        while (nums[i + 1] % nums[i] == 0) {
            nums[i + 1]++;
        }
    }
    for (int i = 0; i < n; i++) {
        cout << nums[i] << " ";
    }
    cout << '\n';
}

int main() {
    // Prepare inputs and call solution
    int i;
    int n;
    vector<int> nums(0);
    
    cin >> i;
    while (i--) {
        cin >> n;
        nums.resize(n);
        for (int count = 0; count < n; count++) {
            cin >> nums[count];
        }
        
        solve(n, nums);
    }
}